lsp-ui.el

```diff
@@ -46,10 +46,10 @@
 (require 'lsp-ui-doc)
 (require 'dash)
 
-(with-eval-after-load 'flycheck
+(when (featurep'flycheck)
```

Comment:
I'm not sure if this works as intended. What if flycheck gets required after lsp-ui ?

Reply:
Yeah, you are right and I don't know a better way to handle this, but, this is a warning from `package-lint` and I gave up to setup it for other problems, I'll undo it too